test/hotspot/jtreg/runtime/cds/appcds/aotClassLinking/BadOldClassA.jasm

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ */
```

Comment:
Header in 2 new tests should be open one, I think.

Reply:
I fixed the copyright.